### PR TITLE
add the ingest_community_events_calender cronjob

### PR DESCRIPTION
**Pull request by hhio618**
##### Description

- Add a cron job to ingest Gitcoin Community events calendar into the marketing.models.upcomingdate model.
- Add the ingest_community_events_calender management command with a url argument set default to [this iCal link](https://calendar.google.com/calendar/ical/7rq7ga2oubv3tk93hk67agdv88%40group.calendar.google.com/public/basic.ics)).

##### Refers/Fixes
Fixes: https://github.com/gitcoinco/web/issues/7499

### CODE REVIEW

File: app/marketing/migrations/0020_auto_20200925_0944.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    dependencies = [
+        ('marketing', '0019_roundupemail_kudos'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='upcomingdate',
+            name='last_modified',
+            field=models.DateTimeField(db_index=True, default=datetime.datetime(2020, 9, 25, 9, 44, 19, 299543)),
```
Comment (hhio618) [inline app/marketing/migrations/0020_auto_20200925_0944.py:17]: Not sure about the default value to populate existing records!

**Comment by owocki** [comment]
cdoe seems fairly sane to me; how did you test this?

**Comment by hhio618** [comment]
Hi @owocki, I have some ideas for test cases. Do we need any particular testes here?
I'll add another commit to the pr.

**Comment by owocki** [comment]
```
92 events are created
0 events are updated
35 events are skipped
DONE
root@48b5e4512227:/code/app# ./manage.py ingest_community_events_calender
/usr/local/lib/python3.6/dist-packages/sorl/thumbnail/conf/__init__.py:16: RemovedInDjango30Warning: The DEFAULT_CONTENT_TYPE setting is deprecated.
  setattr(self, attr, getattr(obj, attr))
System check identified some issues:

WARNINGS:
?: (debug_toolbar.W001) debug_toolbar.middleware.DebugToolbarMiddleware is missing from MIDDLEWARE.
	HINT: Add debug_toolbar.middleware.DebugToolbarMiddleware to MIDDLEWARE.
marketing.UpcomingDate.last_modified: (fields.W161) Fixed default value provided.
	HINT: It seems you set a fixed date / time / datetime value as default for this field. This may not be what you want. If you want to have the current date as default, use `django.utils.timezone.now`
0 events are created
0 events are updated
127 events are skipped
```

amazing!  LGTM

**Review by thelostone-mc — APPROVED** [review]

**Comment by danlipert** [comment]
Hey @hhio618 - looks like the tests were either failing when we merged this PR in, or started failing shortly thereafter. Can you take a look at the tests I removed here and see whats up? Thanks! https://github.com/gitcoinco/web/pull/7727

**Comment by hhio618** [comment]
@danlipert No worries, I'll check it out.

**Pull request merged**